gf-webapp-admin/src/component/rooms/room/apartmentType/ApartmentType.tsx

```diff
@@ -1,23 +1,23 @@
 import {useState} from "react";
-import {Apartments} from "src/model/Room/Apartments";
+import {RoomType} from "src/model/Room/RoomType";
 import {ApartmentsList} from "src/component/rooms/room/apartmentType/apartmentsList/ApartmentsList";
 import {useRoomContext} from "src/component/rooms/room/roomContext";
 import {SmallTitle} from "gf-ui-lib/components/SmallTitle/SmallTitle";
 
 export const ApartmentsType = () => {
   const {room, setRoom} = useRoomContext();
 
-  const [apartmentType, setApartmentType] = useState<Apartments>(room.apartmentsType);
-  const validApartments: string[] = Object.values(Apartments);
+  const [apartmentType, setApartmentType] = useState<RoomType>(room.type);
+  const validApartments: string[] = Object.values(RoomType);
 
-  const isValidApartment = (inputApartment: string): inputApartment is Apartments => {
+  const isValidApartment = (inputApartment: string): inputApartment is RoomType => {
     return validApartments.indexOf(inputApartment) !== -1;
   };
 
   const onChangeValue = (value: string) => {
```

Comment:
bad naming onChangeValue, isValidApartment, value

Reply:
Rename to onChangeRoomType, isValidRoomType, type